### PR TITLE
Upgrade to Python 3

### PR DESCRIPTION
**Pull request by DavidLemayian**
## Description

This PR upgrades Takwimu to use Python 3.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/profiles/census.py [hunk 1]
```diff
@@ -1891,9 +1892,9 @@ def get_worldbank_data(geo, session, country, level):
             cereal_yield_kg_per_hectare, _ = get_stat_data(
                 ['cereal_yield_kg_per_hectare_year', ], geo, session, percent=False)
         except Exception as e:
-            print "\n\n\n\n\n\n\n\n\n\n"
-            print e.message
-            print "\n\n\n\n\n\n\n\n\n\n"
+            print("\n\n\n\n\n\n\n\n\n\n")
+            print(e.message)
```
Comment (kilemensi) [inline takwimu/profiles/census.py]: @phillipahereza the best way of getting error message from python exception is through [`str(e)`](https://stackoverflow.com/a/43453668). `message` will sometime not exist e.g. if you go to Zambia, it will fail. Should be fixed in a separate PR though.

File: docker-compose.yml [hunk 2]
```diff
@@ -27,6 +27,9 @@ services:
       - es
     volumes:
       - "./:/src/takwimu"
+    entrypoint:
+      - python
+    command: ["manage.py", "runserver", "0.0.0.0:8000"]
```
Comment (kilemensi) [inline docker-compose.yml]: These lines killed my morning! Was pulling my hair out "why does everything looks fine in the `Dockerfile` but the takwimu doesn't start?!!"

Sneaky @DavidLemayian. Very sneaky.

Comment (DavidLemayian) [reply docker-compose.yml]: 😂  @kilemensi yeah, this is the better way to do development... particularly allows to edit `takwimu.scss` without having to run collectstatic.

What error are you getting?

Comment (kilemensi) [reply docker-compose.yml]: The `/docker-entrypoint.sh` wasn't running automatically @DavidLemayian. I lost count how many times I rebuilt the images :cry:

> particularly allows to edit `takwimu.scss` without having to run collectstatic.

This sounds cool actually. I'll definitely give it a try it.
